tests/test_devopsUserTest.py

```diff
@@ -0,0 +1,71 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import time
+import unittest
+
+from azure.cli.testsdk import ScenarioTest
+from azure_devtools.scenario_tests import AllowLargeResponse
+from .utilities.helper import disable_telemetry, set_authentication, get_test_org_from_env_variable
+
+DEVOPS_CLI_TEST_ORGANIZATION = get_test_org_from_env_variable() or 'https://dev.azure.com/azuredevopsclitest'
+_TEST_EMAIL_ID = 'new_user_test@outlook.com'
+
+
+class TestUser(ScenarioTest):
+    @AllowLargeResponse(size_kb=3072)
+    @disable_telemetry
+    @set_authentication
+    def test_devops_user_command_addUpdateListShowRemove(self):
+        self.cmd('az devops configure --defaults organization=' +  DEVOPS_CLI_TEST_ORGANIZATION)
+        try:
+            # check user list before adding the user
+            user_list_response = self.cmd('az devops user list -o json --detect off').get_output_in_json()
+            assert len(user_list_response) > 0
+            user_id_found = False
+            for item in user_list_response :
+                if item['user']['mailAddress'] == _TEST_EMAIL_ID:
+                    user_id_found = True
+            assert user_id_found == False
+
+            #add user
+            user_add_response = self.cmd('az devops user add -o json --detect off --user ' + _TEST_EMAIL_ID + ' --license-type stakeholder').get_output_in_json()
```

Comment:
can we add early adopter here also?
That way this test will be able to run on non-microsoft orgs as well?

Reply:
EarlyAdopter can't be assigned manually for microsoft account. And other license types won't be updated for microsoft account. Hence, removing the license types related checks so that these tests run successfully for all accounts/users.